apps/cowswap-frontend/src/legacy/state/orders/utils.ts

```diff
@@ -414,6 +406,19 @@ export function getRemainderAmountsWithoutSurplus(order: Order | ParsedOrder): {
   }
 }
 
+function getSurplusAmountBigNumber(order: Order | ParsedOrder): BigNumber {
+  let surplusAmountBigNumber: BigNumber
```

Comment:
Excessive let, we can do just return in if and else.
p.s. I'm immutable code nazi :)